### DISCUSSION TITLE
Transparent Menu over Hero Slider

### DISCUSSION THREAD

**Discussion by Fedora-Man** [holger1411/understrap#1604] [Ideas]
Hi,

How to make the Main Menu (navbar) transparent over the Hero slider?

Best Regards,

Hasan

**Comment by bacoords** [comment]
Basically. you'll need to do a few things: 
1. Make sure you have a transparent background on your navbar, probably with `.bg-transparent`
2. Add a negative bottom margin to your `#wrapper-navbar`. maybe `margin-bottom:-90px` will work but I recommend testing at all screen sizes
3. Check your z-index - your header may end up "under" your hero so play around with z-indexes on the #wrapper-navbar and the hero area

**Comment by Fedora-Man** [comment]
I did it on the #main-nav level instead of #wrapper-navbar so it applies to the logo and the button:

#main-nav {
    margin-bottom: -152px;
    z-index: 100;
}

Thanks
